bitnami/elasticsearch/templates/data-statefulset.yaml

```diff
@@ -178,6 +183,14 @@ spec:
               name: "config"
               subPath: elasticsearch.yml
             {{- end }}
+            {{- if .Values.keystore }}
+            - name: keystore
+              emptyDir: {}
+            {{- range .Values.keystore }}
+            - name: keystore-{{ .secretName }}
+            {{- end}}
+          secret: {{ toYaml . | nindent 12 }}
```

Comment:
Could we use here the tplValues function?

Reply:
Where is the keystore?